tests/unit/constraints/test_tabular.py

```diff
@@ -1201,24 +1246,25 @@ def test_reverse_transform_floats(self):
         transformed = pd.DataFrame({
             'a': [1.1, 2.2, 3.3],
             'c': [7, 8, 9],
-            'a#b': [np.log(4)] * 3,
+            'a#b': [np.log(4), 0, -10],
         })
         out = instance.reverse_transform(transformed)
 
         # Assert
         expected_out = pd.DataFrame({
             'a': [1.1, 2.2, 3.3],
             'c': [7, 8, 9],
-            'b': [4.1, 5.2, 6.3],
+            'b': [4.1, 2.200001, 3.300001],
         })
-        pd.testing.assert_frame_equal(out, expected_out)
+        pd.testing.assert_frame_equal(out, expected_out, rtol=1e-8)
```

Comment:
the default rtol=1e-6 is not precise enough to test the numbers (the test always passes, even when the data is incorrect.)